pinot-common/src/main/proto/worker.proto

```diff
@@ -70,23 +70,17 @@ message QueryResponse {
 
 message StagePlan {
   int32 stageId = 1;
-  string instanceId = 2;
+  string virtualAddress = 2;
   StageNode stageRoot = 3;
-  map<int32, StageMetadata> stageMetadata = 4;
+  repeated StageMetadata stageMetadata = 4;
 }
 
 message StageMetadata {
```

Comment:
Also, a stage having multiple stageNode is a bit confusing.. should we call this StagePlan?

StageMetadata is really StagePlan Metadata.

Reply:
this is a legacy issue that needs to be clean up next --> right now the stageMetadata list is including all the stages in the request. 

next follow up is to only send the stageMetadata with resolved mailbox info (right now mailbox info is resolved on server)